docs/openapi.yaml

```diff
@@ -1159,7 +1159,7 @@ components:
           allOf:
           - $ref: '#/components/schemas/model.RemoveFilesType'
           description: "Whether to clear the output directory (default: KeepAll)."
-          type: object
+          type: boolean
```

Comment:
actually, it should be string, with KeepAll,RemoveAll,RemoveIncremental as possible values.

Reply:
It is an enum in the spec and should have the object type.